src/Tec/Settings.php

```diff
@@ -274,19 +295,48 @@ function ( $key ) {
 	/**
 	 * Here is an example of getting some HTML for the Settings Header.
 	 *
-	 * TODO: Delete this method if you do not need a heading for your settings. Also remove the corresponding element in the the $fields array in the `add_settings()` method above.
-	 *
 	 * @return string
 	 */
-	private function get_example_intro_text() {
-		$result = '<h3>' . esc_html_x( 'Example Extension Setup', 'Settings header', '__TRIBE_DOMAIN__' ) . '</h3>';
+	private function get_defaiult_fieldset_intro_text() {
+		$result = '<h3>' . esc_html_x( 'Default Ticket Fieldsets for Collecting Attendee Registration Information', 'Settings header', 'tec-labs-default-ticket-fieldset' ) . '</h3>';
 		$result .= '<div style="margin-left: 20px;">';
 		$result .= '<p>';
-		$result .= esc_html_x( 'Some text here about this settings section.', 'Setting section description', '__TRIBE_DOMAIN__' );
+		$result .= esc_html_x( 'You can set up default fieldsets that will be saved with every newly created RSVP or ticket, for tickets created both on the backend or through the Community Events submission form.', 'Setting section description', 'tec-labs-default-ticket-fieldset' );
+		$result .= ' ';
+		$result .= esc_html_x( 'If a fieldset is already being added to a ticket manually, then the defaults will not be applied, unless the override setting is enabled.', 'Setting section description', 'tec-labs-default-ticket-fieldset' );
+		$result .= '<br>';
+		$result .= sprintf(
+			// Translators: %1$s opening <a> tag with URL, %2$s closing </a> tag
+			esc_html_x(
+				'You can create ticket fieldsets for attendee information collection %1$shere%2$s.',
+				'Setting section description',
+				'tec-labs-default-ticket-fieldset'
+			),
+			'<a href="/wp-admin/edit.php?post_type=ticket-meta-fieldset">',
```

Comment:
Do we want to use the relative URL here? This can bring trouble if they have a strange directory install, or multisite. I'd rather use the absolute URL.

Reply:
Thanks, I was expecting that. I'm not sure how to get that but will look into it.